universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -66,6 +85,8 @@ private ContainerTag buildEditQuestionForm(QuestionDefinition definition) {
   private ContainerTag buildQuestionForm(QuestionForm questionForm) {
     ContainerTag formTag = form().withMethod("POST");
     formTag
+        .with(hiddenInputWithValue("questionId", questionForm.getQuestionIdString()))
+        .with(hiddenInputWithValue("questionVersion", questionForm.getQuestionVersionString()))
```

Comment:
Suggestion (blocking): recommend removing this because the version isn't necessary for updating the question